README.md

```diff
@@ -1,86 +1,99 @@
-- Install the following packages: libsdl2-dev cmake build-essential git libboost-locale-dev libboost-filesystem-dev libboost-system-dev libboost-program-options-dev
-```
+
+(Tested on Ubuntu 16.04)
+
+* Install the following packages:
+
+```sh
 sudo apt-get install libsdl2-dev cmake build-essential git libunwind8-dev libboost-locale-dev libboost-filesystem-dev libboost-system-dev libboost-program-options-dev
 ```
-- Checkout OpenApoc from github
-- Fetch the dependencies from git with the following terminal command (run from the just-created OpenApoc folder)
-```
+
+* Checkout OpenApoc from GitHub.
+* Fetch the dependencies from git with the following terminal command (run from the just-created OpenApoc folder).
+
+```sh
 git submodule init
 git submodule update
 ```
-- Copy the cd.iso file to the 'data' directory under the repository root (Note - despite dosbox having good linux support, the steam version of X-Com Apocalypse refuses to install in steam for linux - you may need to snatch the cd.iso file off a windows steam install)
-```
+
+* Copy the cd.iso file to the 'data' directory under the repository root (Note - despite dosbox having good linux support, the steam version of X-Com Apocalypse refuses to install in steam for linux - you may need to snatch the cd.iso file off a windows steam install).
+
+```sh
 cp /path/to/cd.iso data/
 ```
-- Create a subdirectory ('build' in this example) in the OpenApoc checkout directory, and from that use cmake to configure OpenApoc:
-```
+
+* Create a subdirectory ('build' in this example) in the OpenApoc checkout directory, and from that use cmake to configure OpenApoc.
+
+```sh
 cd /path/to/OpenApoc
 mkdir build
 cd build
 cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
 ```
-- This cmake command will fail if we're missing a dependency, or your system is for some other reason unable to build - if you have any issues please contact us (see above for links)
-- Build the project with the following command
-```
+
+* This cmake command will fail if we're missing a dependency, or your system is for some other reason unable to build - if you have any issues please contact us (see above for links).
+* Build the project with the following command.
+
+```sh
 make -j4
 ```
-- This should create a directory 'bin' under the build directory, with the 'OpenApoc' executable file. OpenApoc by default expects the data folder to be in the current working directory, so running the executable from the root of the git checkout should work.
-```
+
+* This should create a directory 'bin' under the build directory, with the 'OpenApoc' executable file. OpenApoc by default expects the data folder to be in the current working directory, so running the executable from the root of the git checkout should work.
+
+```sh
 ./build/bin/OpenApoc
 ```
+
+## Contact us
+
+If you're interested, please visit our [website](http://openapoc.org).
+We also have [forums](http://openapoc.pmprog.co.uk) - please pop by and introduce yourself!
```

Comment:
This has changed to openapoc.org too - though openapoc.pmprog.co.uk forwards there right now, we can't rely on that forever